app/src/main/java/tech/ula/utils/LogUtility.kt

```diff
@@ -19,4 +19,10 @@ class LogUtility {
     fun d(tag: String, message: String) {
         Log.d(tag, message)
     }
+
+    fun runtimeErrorForCommand(functionName: String, command: String, err: Exception) {
```

Comment:
I still prefer a more action-oriented naming for convention type personally. Something like `fun logRuntimeErrorForCommand()`.

Reply:
I just noticed the codebase has function names that all start with a verb.  I'll stick with that theme.